api/config/config_perf.json

```diff
@@ -13,7 +13,7 @@
     "billingRetryCount": 4,
     "billingProjectBufferCapacity": 20,
     "xAppIdValue": "perf-AoU-RW",
-    "vpcServicePerimeterName": "accessPolicies/357236995176/servicePerimeters/terra_prod_aou_perf"
+    "vpcServicePerimeterName": "TODO"
```

Comment:
nit: would just omit this parameter entirely and leave the TODO in a comment if needed

Reply:
I think the build will complain if its completely omitted.

Reply:
I don't think it will. But if you're concerned about it, I'll also accept empty string. I just don't want "TODO" in a live code path on principle.